lib/bootloader_spvm.pm

```diff
@@ -55,6 +68,57 @@ sub get_into_net_boot {
     assert_screen ["pvm-grub", "novalink-failed-first-boot"];
 }
 
+=head2 boot_spvm
+
+ boot_spvm();
+
+Boot from spvm backend via novalink.
+
+Detach possibly attached terminals by command C<rmvterm --id $lpar_id>.
+$lpar_id is 'Logical Partition ID' of PowerVM.
+
+Power off the machine if it's still running by C<pvmctl lpar power-off -i id=$lpar_id --hard>.
+
+Use console 'novalink-ssh' to run yast installer later.
+
+Make sure that the default boot mode is 'Normal' and not 'System_Management_Services'.
+
+Boot LPAR to SMS for default boot mode 'Normal' and establish ssh connection by C<mkvterm --id $lpar_id>.
+
+Run C<get_into_net_boot()>.
+
+Reset LPAR manually at booting stage to firmware if it was reconfigured in between and the first menu entry was 
+used to enter the command line.
+
+Try to wait 3 times for prompt issue and clear the prompt (and create an error) in case the above went wrong
+
+Type for yast installer: 
+
+=over
+
+* repository $repo
+
+* mirror $mirror
+
+* mount point $mntpoint
+
+* bootmenu_default_params
+
+* bootmenu_network_source
+
+* specific_bootmenu_params
+
+* registration_bootloader_params
+
+* remote_install_bootmenu_params
+
+=back
+
+Delete partition table before starting installation.
+
+Switch to installation console (ssh or vnc).
```

Comment:
Here too:
I think this is too verbose. The code is already clarifying this. This description would be enough with **what** is done and **why** is done. Maybe just specifying anything spvm specific that needs to be known in advance to understand the code.